relay-metrics/src/aggregation.rs

```diff
@@ -1803,28 +1784,18 @@ impl Aggregator {
             let partitioned_buckets = self.partition_buckets(project_buckets, num_partitions);
             for (partition_key, buckets) in partitioned_buckets {
                 self.process_batches(buckets, |batch| {
-                    let fut = self
-                        .receiver
-                        .send(FlushBuckets {
-                            project_key,
-                            partition_key,
-                            buckets: batch,
-                        })
-                        .into_actor(self)
-                        .and_then(move |result, slf, _ctx| {
-                            if let Err(buckets) = result {
-                                relay_log::trace!(
-                                    "returned {} buckets from receiver, merging back",
-                                    buckets.len()
-                                );
-                                slf.merge_all(project_key, buckets).ok();
-                            }
-                            fut::ok(())
-                        })
-                        .drop_err();
-
-                    if let Some(context) = context.as_deref_mut() {
-                        fut.spawn(context);
+                    let batch_size = batch.len() as u64;
+                    if let Err(err) = self.receiver.do_send(FlushBuckets {
+                        project_key,
+                        partition_key,
+                        buckets: batch,
+                    }) {
```

Comment:
_nit_: For readability, I personally prefer not to break lines in conditions and instead assign to a temporary.

```suggestion
                    let result = self.receiver.do_send(FlushBuckets {
                        project_key,
                        partition_key,
                        buckets: batch,
                    });
                    
                    if let Err(err) = result {
```

Reply:
addressed in [498277e](https://github.com/getsentry/relay/pull/1492/commits/498277ea80d283500dd24e56d90a3b2fd2c6b3ed)